media-libs/kimageannotator/kimageannotator-0.4.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VIRTUALX_REQUIRED="test"
+inherit cmake virtualx
+
+MY_PN=kImageAnnotator
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Tool for annotating images"
+HOMEPAGE="https://github.com/ksnip/kImageAnnotator"
+SRC_URI="https://github.com/ksnip/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
```

Comment:
If the SONAME change is breaking revdeps, this should specify a subslot. 
https://devmanual.gentoo.org/general-concepts/slotting/index.html#sub-slots

@a17r please take it from here? :)

Reply:
no need to, as it is a fix, and the only revdep can be bumped in lockstep with a new version requirement.